.github/workflows/pull-request.yaml

```diff
@@ -26,24 +26,24 @@ jobs:
         run: sudo make pull-scripts
 
       - name: Check release.yaml
-        run: make check-release-yaml
+        run: sudo make check-release-yaml
```

Comment:
I'm confused as to why the `sudo` is needed. I don't see anything that requires it - `scripts/check-release-yaml` accesses `release.yaml` and uses `yq` and `git`, none of which should require `sudo`. Is this a leftover from debugging or something?

Reply:
This change actually comes from: https://github.com/rancher/charts/pull/2955/files

This branch probably was outdated because it was created from the previous branch before these changes.